strongswan/prefix-log

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# setup fd-3 to point to the original stdout
```

Comment:
Would be nice with a short description of what to expect from this file. Any specific reason for writing this in perl instead of bash as we usually use for scripting?

Reply:
Only reason is it is stolen from somebody. Will write a section with a reference 👍

Reply:
Description added 👍